src/util.ts

```diff
@@ -76,6 +76,31 @@ export async function fileExists(filePath: string) {
     });
 }
 
+/**
+ * Reads the the manifest file and converts to a javascript object skipping empty lines and comments
+ * @param path location of the manifest file
+ */
+export async function convertManifestToObject(path: string): Promise<{ [key: string]: string } | undefined> {
+    return new Promise(async (resolve) => {
```

Comment:
Since this is already an async function, you don't need to wrap this in a promise. Just `return undefined`, or `return manifestValues`.